lib/package.gd

```diff
@@ -691,6 +691,31 @@ DeclareGlobalFunction( "LoadPackageDocumentation" );
 ##
 ##  <Description>
 ##  loads the &GAP; package with name <A>name</A>.
+## <P/>
+##  As an example, the following loads the &GAP; subgroups package
```

Comment:
I suggest to use `loads the &GAP; package <Package>SONATA</Package> which provides methods for the construction and analysis of finite nearrings:`